tools/lkl/tests/hijack-test.sh

```diff
@@ -1,13 +1,31 @@
-#!/bin/bash -e
+
+#!/bin/bash
+
+set -e
 
 script_dir=$(cd $(dirname ${BASH_SOURCE:-$0}); pwd)
-hijack_script=lkl-hijack.sh
-export PATH=${script_dir}/../bin/:${PATH}
+hijack_script=${script_dir}/../bin/lkl-hijack.sh
 
-if ! [ -e ${script_dir}/../liblkl-hijack.so ]; then
+if ! [ -e ${script_dir}/../lib/liblkl-hijack.so ]; then
```

Comment:
@thehajime Good catch -- sorry I missed that. I made this branch before that change went through, I suppose. It looks like it should all be good now.